Dockerfile

```diff
@@ -9,3 +13,6 @@ RUN cargo build -p mc-reserve-auditor --release
 
 FROM mobilecoin/runtime-base:sha-6f1c083
 COPY --from=builder /build/target/release/mc-reserve-auditor /usr/local/bin/mc-reserve-auditor
+COPY --from=mobilecoind /usr/bin/mobilecoind /usr/bin/mobilecoind
+
+CMD ["/usr/bin/supervisord"]
```

Comment:
Is this needed as we're passing in cmd supervisord -n for the mobileocind-ledger-scanner container and starting http-server with /bin/bash